tools/pipeline-runner/cloudbuild-test.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+# Load secrets from secret manager
+- name: 'gcr.io/$PROJECT_ID/devrel-pipeline:latest'
+  entrypoint: "sh"
+  args:
+    - "-c"
+    - |-
+      echo "export APIGEE_PASS=\"$(gcloud secrets versions access latest \
```

Comment:
note to self, move this into a standalone shell script at some point

Reply:
noted in #156